lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const ember = require('../utils/ember');
+const utils = require('../utils/utils');
+
+
+//------------------------------------------------------------------------------
+// Ember object rule - Avoid leaking state
+// (Don't use arrays or objects as default props)
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Avoids state leakage',
+      category: 'Ember Object',
+      recommended: true
+    },
+    fixable: null, // or "code" or "whitespace"
+    schema: [{
+      type: 'array',
+      items: { type: 'string' },
+    }],
+  },
+
+  create(context) {
+    const defaultIgnoredProperties = [
```

Comment:
This array can live above the `module.exports` line (in module scope) since it is static.

Reply:
We should probably change the case too -- `DEFAULT_IGNORED_PROPERTIES`